src/Microsoft.AspNetCore.Mvc.ViewFeatures/Resources.resx

```diff
@@ -280,4 +280,7 @@
   <data name="ValueInterfaceAbstractOrOpenGenericTypesCannotBeActivated" xml:space="preserve">
     <value>The type '{0}' cannot be activated by '{1}' because it is either a value type, an interface, an abstract class or an open generic type.</value>
   </data>
+  <data name="ViewData_CannotCreateOfTModelType" xml:space="preserve">
+    <value>Cannot find matching constructor to create ViewDataDictionary of type '{0}'.</value>
+  </data>
```

Comment:
When will this error ever be surfaced? This seems like it should just be a debug.assert.